packages/elements/src/combo-box/__test__/combo-box.value.test.js

```diff
@@ -76,7 +76,42 @@ describe('combo-box/Value', () => {
       expect(el.value).to.equal('Any', 'Value must be "Any" string');
     });
 
-    it('Free text. Reset value via API', async () => {
+    it('Free text: Set any value via API then select value in the list', async () => {
```

Comment:
I've moved "Free Text mode" test cases into their own `describe`. It might not be perfect but revisiting test case organization is beyond the scope of this PR.